docker/celery/entrypoint.sh

```diff
@@ -8,10 +8,6 @@ poetry run -C $HOME/ python3 manage.py loaddata fixtures/default_scan_engines.ya
 poetry run -C $HOME/ python3 manage.py loaddata fixtures/default_keywords.yaml --app scanEngine.InterestingLookupModel
 poetry run -C $HOME/ python3 manage.py loaddata fixtures/external_tools.yaml --app scanEngine.InstalledExternalTool
 
-# Compile messages translations
-find . -type f -name "*.po" -exec sed -i 's/^#~ //g' {} +
-poetry run -C $HOME/ python3 manage.py compilemessages
-
```

Comment:
I thought poetry was part of the improvements? Shouldn't this remain? Or is it deleted because of the introduction of `pyproject.toml`?

Reply:
It's remove because @Talanor left this by error, I've asked to him.
Those 3 lines are for the internationalization part which is not implemented yet